Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -5084,12 +5095,17 @@ namespace AzToolsFramework
 
         for (auto componentEditor : componentEditors)
         {
-            AzToolsFramework::ReflectedPropertyEditor::WidgetList widgets = componentEditor->GetPropertyEditor()->GetWidgets();
-            for (auto& [dataNode, rowWidget] : widgets)
+            // May or may not need to handle this for the DPE
```

Comment:
Does this need to be reworded? makes me think the code may or may not work

Reply:
Yup, I removed it since it was just for me to remember to address it in the IPropertyEditor